src/privatesend-client.cpp

```diff
@@ -697,12 +697,12 @@ bool CPrivateSendClient::DoAutomaticDenominating(bool fDryRun)
         return false;
     }
 
-    CAmount nValueMin = vecPrivateSendDenominations.back();
+    CAmount nValueMin = CPrivateSend::GetSmallestDenomination();
 
     // if there are no confirmed DS collateral inputs yet
     if(!pwalletMain->HasCollateralInputs()) {
         // should have some additional amount for them
-        nValueMin += PRIVATESEND_COLLATERAL*4;
+        nValueMin += CPrivateSend::GetCollateralAmount() * 4;
```

Comment:
'* 4' is used with collateral amount many times in different files, why not define it as constant or entirely as:
CPrivateSend::GetCollateralMaxAmount()